shapes/README.md

```diff
@@ -1,106 +1,235 @@
+# Solid User Profile
+
+We don't currently have a specific Shape defined for user profiles. But the
+original Solid specification provides good guidance on the basic requirements
+for the Profile Document expected to be returned when de-referencing a WebID.
+
+See here for details: https://github.com/solid/solid-spec#webid-profile-documents
+
+We'd certainly be open to ideas on how to model these basic discovery
+requirements differently, and also on how to model the actual human user profile
+information (such as date-of-birth, gender, address, etc.).
+
+## The 'country' problem?
+An interesting discussion point would be how to model something as seemingly
+trivial as the 'country' field of a user's address. If we take the ISO as just
+an example of a recognized 'authority' when referring to "countries of the
+world" (of course they are by no means the only such authority), then a problem
+arises in that the ISO do not provide IRIs for the countries they otherwise
+standardize identifiers for (e.g., 'ie' for Ireland, 'de' for Germany).
+
+(About 7 years ago I specifically asked the ISO if they might consider minting
+official IRIs for countries. Their reply was, literally, "**_We do not product
```

Comment:
```suggestion
official IRIs for countries. Their reply was, literally, "**_We do not produce
```

Reply:
:) That's not a typo (at least not in my post) - that's what the reply from the ISO was, **_literally_**! (I suppose I could add (sic) after it though...)

Reply:
^^, well, that adds salt to the answer